### PR TITLE
Create unit test for VAOS appointment helpers (utility/appointment.jsx) #4890

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
Create unit test for VAOS appointment helpers (utility/appointment.jsx)

## Testing done
Unit test

## Screenshots
N/A

## Acceptance criteria
- [ ] All unit test should pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/utils/appointment.unit.spec.jsx
Comment (jbalboni) [inline src/applications/vaos/tests/utils/appointment.unit.spec.jsx]: You'll probably have to move this and the other new tests back to your other ticket.

**Review by jbalboni — APPROVED** [review]

File: src/applications/vaos/tests/utils/appointment.unit.spec.jsx
Comment (vbahinwillit) [reply src/applications/vaos/tests/utils/appointment.unit.spec.jsx]: I think I did already.

**Review by jbalboni — APPROVED** [review]

**Review by bkjohnson — COMMENTED** [review]
Since this overlaps a lot with your other PR I'll wait until any merge issues have been resolved before approving.

**Pull request closed without merging**
